blend/navbars.less

```diff
@@ -316,6 +318,18 @@ titlebar {
         letter-spacing: -1px;
         margin: 0;
 
+        .title-text {
+          position: absolute;
+          width: calc(100%-15px);
```

Comment:
interestingly, this isn't exactly what you want. the LESS compiler will compile this to:
width: calc(85%);

what you want to put here is: 
width: calc(~"100% - 15px"); 

the tilde and quotes mean that the compiler won't try to perform its math on them. you also need the spaces for it to be valid. finally, i think you're actually going to want around ~45 px of space to the right for it to look good. ~20px for the icon, and another ~25px of padding.